docs/_steps/workflow_steps_overview.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Overview of Workflow Steps for apps
+title: Overview of Workflow Steps for apps <span class="label-legacy">LEGACY</span>
```

Comment:
Might be nice to add a disclaimer on this page on why we're marking this as legacy. Maybe something like:

Workflow Steps from Apps are a legacy feature, not to be confused with Workflows that are part of the next-gen beta. They are not interchangeable features. We encourage those who are currently publishing Workflow Steps from apps to join the beta.  

@stevengill would it make sense in a next docs pass to add a  "migrating a workflow step from apps to next-gen workflow" section of the docs....